.github/ISSUE_TEMPLATE/release_new_version.md

```diff
@@ -21,14 +21,18 @@ git push -u origin release
 
 **AFTER PRODUCT OWNER APPROVAL**
 
-- [ ] Draft new release
-  - [ ] Tag release using `vX.Y.Z` format
+- [ ] Create a Tag on the release branch with the release version using the `vX.Y.Z` format
```

Comment:
please revert to initial version. Tag will be created when drafting the release and should not be pushed manually. Let's not expose such implementation details.

Reply:
I rephrased it. So it is clear that the tag will be created when the release is published.